### ISSUE TITLE
The question in loss function yolo_loss 

### ISSUE THREAD

**Issue by MoJiMin** [qqwweee/keras-yolo3#99]
could you please tell what is the mean of box_loss_scale in yolo_loss?
and I can‘t’ find the coefficient “λcoord ” and “λnoobj” , could you tell me where are they?
Thanks!

**Comment by qqwweee** [comment]
box_loss_scale is to adjust the importance of box loss. The smaller the box is, the larger the scale is. For a true box with wh (0.5, 0.6), the scale is 2-0.5\*0.6=1.7.
There is no “λcoord ” and “λnoobj” because I didn't see them used in Darknet YOLOv3. Actually, they can be seen as hyperparameters. Add them if you like.

**Comment by linydf** [comment]
@MoJiMin if you add this to your code?this is the result that my train loss is big about 39.and test result is poor???
